src/Validator/Refund/RefundUnitsCommandValidator.php

```diff
@@ -47,6 +47,10 @@ public function validate(RefundUnits $command): void
             throw OrderNotAvailableForRefunding::withOrderNumber($command->orderNumber());
         }
 
+        if (empty($command->units()) && empty($command->shipments())) {
+            throw new OrderNotAvailableForRefunding(sprintf('There is no units to refund in order %s', $command->orderNumber()));
```

Comment:
Is this a public exception? If so, we should translate the message

Reply:
It's not public, but I found typo